changelog.md

```diff
@@ -10,6 +10,7 @@
 - change the signature of `Uploader->fromUrl()`, old signature is deprecated but will work until 3.0
 - deprecate `File->copy()` and `File->copyTo()`
 - deprecate `Api->copyFile()`
+- methods: File::crop & File::scaleCrop throws exceptions if parameters $width or $height are 0 or not provided.
```

Comment:
well, this one is released already, make a new section `2.1.0-pre` or something, also please use consistent convention, e.g. `File->crop()` instead of File::crop
if you feel that your way is better, let's discuss and change the whole document